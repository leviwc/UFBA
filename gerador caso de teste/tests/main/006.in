29 391
12 25
23 4
22 25
25 17
15 15
9 13
5 2
15 21
4 26
1 4
15 9
12 19
11 11
26 17
18 9
2 17
17 14
14 26
19 23
20 9
20 15
12 10
29 24
28 8
4 5
1 1
6 16
8 23
21 24
14 4
26 12
1 22
7 18
15 19
16 21
19 24
14 2
15 11
23 2
24 11
28 24
13 29
13 2
1 14
17 10
10 22
19 18
10 8
26 5
6 14
28 13
19 29
5 22
28 26
21 21
13 10
21 8
11 21
13 5
14 29
16 23
21 4
20 1
25 27
3 29
19 2
15 6
11 13
9 29
20 23
12 11
11 5
20 5
12 12
21 22
27 14
3 28
19 7
16 19
27 6
23 13
8 2
1 21
2 9
26 3
8 14
17 8
6 23
12 6
28 2
28 6
24 23
28 12
19 8
27 20
3 7
24 7
5 8
20 21
21 10
16 28
28 29
6 4
19 13
8 4
3 18
9 1
13 8
7 11
26 13
21 13
21 2
28 21
18 18
21 3
1 23
5 21
15 16
5 18
6 1
28 20
5 15
14 11
12 16
6 24
27 8
24 14
15 1
25 24
9 12
21 25
28 18
15 18
27 1
16 25
6 11
17 7
14 23
23 11
22 29
3 23
21 18
28 19
28 25
26 2
18 4
12 20
28 4
4 29
19 20
4 16
28 5
9 27
13 22
24 3
1 18
1 24
6 8
8 16
17 11
26 27
28 17
23 5
25 5
24 26
1 10
24 2
3 22
19 3
9 9
11 22
7 28
8 1
25 20
14 14
19 10
20 16
10 9
12 5
25 6
17 1
10 3
21 29
28 11
17 12
9 28
13 25
18 11
20 2
5 17
13 1
4 9
7 2
19 14
2 27
10 6
12 18
24 18
7 7
23 15
9 7
21 23
17 22
2 18
10 15
10 20
8 12
22 20
3 12
12 15
13 14
9 14
27 21
22 9
12 29
9 6
22 15
5 5
10 28
18 13
17 6
10 7
5 14
29 18
24 17
9 19
10 25
6 13
28 15
16 27
23 9
8 11
21 17
7 23
8 22
7 16
29 16
28 1
5 1
16 5
1 25
11 9
14 28
12 23
26 1
3 2
14 25
18 26
2 1
5 29
26 7
4 17
14 18
3 17
13 3
15 29
19 4
8 7
27 24
4 10
24 8
8 26
5 6
18 23
20 24
22 18
18 27
10 16
27 3
5 7
10 23
2 15
12 7
29 17
12 2
8 9
2 10
29 26
7 6
17 27
21 14
17 23
9 26
10 18
18 8
17 20
26 26
12 1
29 20
4 15
18 6
27 28
22 24
4 3
16 3
14 10
3 14
1 16
2 11
6 2
27 13
19 19
17 18
6 3
11 16
11 4
19 11
26 22
20 20
15 14
20 7
11 1
8 3
19 26
14 20
24 4
20 11
23 25
15 26
25 3
6 21
2 22
15 24
16 13
29 2
25 8
19 17
23 26
2 4
19 6
22 23
20 6
9 5
11 27
13 12
7 25
3 3
14 7
6 22
25 18
20 3
25 19
11 10
8 15
1 29
22 19
3 15
3 9
4 13
14 16
26 25
24 9
17 15
28 23
28 28
22 12
7 29
27 27
25 29
5 27
22 4
18 20
10 10
14 12
6 6
24 24
4 20
27 7
16 17
27 23
17 9
19 5
20 26
29 29
3 11
7 13
25 2
17 13
6 26
7 21
5 24
12 21
29 11
19 27
15 7
29 27
20 8
9 21
24 10
9 16
27 22
