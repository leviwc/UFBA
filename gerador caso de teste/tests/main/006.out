Resultado com busca binaria: 3
3200.236000 s
Resultado com backtracking apenas: 3
0.024000 s
Resultado testando todas as possibilidades: 3
17996.419000 s
Resultado com busca binaria otimizada: 3
60.243000 s
